wallet/src/account/output_cache/mod.rs

```diff
@@ -1173,6 +1173,57 @@ impl OutputCache {
             .collect()
     }
 
+    pub fn confirmed_transactions(
+        &self,
+        destination: Option<Destination>,
+        limit: usize,
+    ) -> Vec<WithId<&Transaction>> {
+        self.txs
```

Comment:
Is it possible to order these transactions in descending order, from newest to oldest?